Makefile

```diff
@@ -0,0 +1,28 @@
+CONTAINER_ENGINE ?= podman
+
+CHECKUP_IMAGE_NAME := quay.io/kiagnose/kubevirt-rt-checkup
+CHECKUP_IMAGE_TAG ?= devel
+
+GO_IMAGE_NAME := docker.io/library/golang
```

Comment:
dockerhub is known to limit pulls, this may become a problem later.

Reply:
Yes I am aware of this rate limit.
I have found a mirror in quay, but it is not very up-to-date with the official one:
https://quay.io/repository/projectquay/golang?tab=tags